code/modules/admin/sql_message_system.dm

```diff
@@ -219,6 +232,48 @@
 			browse_messages(target_ckey = target_ckey, agegate = TRUE)
 	qdel(query_find_edit_expiry_message)
 
+/proc/edit_message_severity(message_id, browse)
+	if(!SSdbcore.Connect())
+		to_chat(usr, "<span class='danger'>Failed to establish database connection.</span>")
+		return
+	message_id = text2num(message_id)
+	if(!message_id)
+		return	
+	var/datum/DBQuery/query_find_edit_note_severity = SSdbcore.NewQuery("SELECT type, targetckey, adminckey, severity FROM [format_table_name("messages")] WHERE id = [message_id] AND deleted = 0")
```

Comment:
Just copying the subqueries from i.e. expiration edit proc will work to replace `targetckey, adminckey`